docs/CHANGELOG.md

```diff
@@ -45,17 +45,28 @@ recommended as a matter of style.  This is only possible if the upper bound type
 qualifier is the top type qualifier for every type system; otherwise, some
 annotations must remain and so must `extends Object`.
 
-For wildcards, the same transformations apply, with one simplification:  instead
-of `? extends @BottomType Object`, you can write `@BottomType ?`, which is
-shorter and may be easier to read.
+There is one simplification:  instead of `<T extends @BottomType Object>`, you
+can write `<@BottomType T>, which is shorter and may be easier to read.
 
-For the Nullness Checker in particular, `T extends Object` should be changed to
-`@NonNull T` or `T extends @NonNull Object`, and `? extends Object` should be
-changed to `@NonNull ?`.
+For wildcards, the same transformations apply.
+
+For the Nullness Checker in particular, `<T extends Object>` should be changed to
+`<@NonNull T>` or `<T extends @NonNull Object>`, and `<? extends Object>` should be
+changed to `<@NonNull ?>`.  `<T extends @Nullable Object>` and `<@Nullable T>` can be
+changed to `<T>`, and `<? extends @Nullable Object>` and `<@Nullable ?>` can be changed to
+`<?>`.
 
 No code changes are required for type systems that default explicit and implicit
 bounds the same, such as the Locking Checker.
 
+Here is how to find places that your code might need to be changed.  Call a
+"non-top-default type system" one of the ones at listed at
+https://checkerframework.org/manual/#default-is-not-top-type-systems.  Consider every
+occurrence of "`extends Object`":
+ * that is in a file that contains `@AnnotatedFor` for a non-default-top type system, or
```

Comment:
`@AnnotatedFor` should not be used in code that a type checker is run on.   But maybe this whole section is about updating an annotated library?